docs/developerguide.rst

```diff
@@ -9,4 +9,43 @@ understand REANA source code and contribute to it.
 Debugging
 ---------
 
-We use ``wdb``.
+Assuming that REANA is being run in development mode following
```

Comment:
Instead of "Assuming...", you can use walk-through mode tone, such as:

"In order to debug a REANA component, you first have to install REANA cluster in the development mode.  You do this as follows...":